validator/testcases/regex.py

```diff
@@ -334,6 +362,23 @@ def rewrap():
 
 
 @register_generator
+class ExtensionManagerRegexTests(RegexTestGenerator):
+    """
+    Tests for uses of the old extension manager API, which should not be
+    referenced in new extensions.
+    """
+
+    def js_tests(self):
+        yield self.get_test(
+                r"@mozilla\.org/extensions/manager;1|"
```

Comment:
Yeah, I guess leave it like that, at least for now, maybe one day i'll do a "flake8 pass" on all of amo-validator. Thanks @kmaglione